Corpsman Glasses

Unkn0wnGh0st333 opened the pull request:
<!-- Please read these guidelines before opening your PR: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
<!-- The text between the arrows are comments - they will not be visible on your PR. -->

## About the PR
Added Corpsman Glasses to the game, added it to Corpsman's lockers, and added a crafting recipe for them.

## Why / Balance
The Corpsman gets flashed at least 2 billion times per game. This is basically a SecMed hud with flash protection, something all Corpsmen dream of.

## Technical details
Added Corpsman Glasses, Replaced Security Glasses with Corpsman Glasses in Corpsman lockers, Added crafting recipe for Corpsman Glasses based off of the Security Glasses except using a MedSecHud. Added MedSecHuds to the Tags list to be used in crafting.

## Media
![image](https://github.com/user-attachments/assets/eaa8afab-cf84-4104-9776-0a53a1d8f465)
![image](https://github.com/user-attachments/assets/10171a84-914a-4a7e-b9d9-454d5414c9ba)

## Requirements
- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
¯\_(ツ)_/¯

**Changelog**
🆑 
- Added Corpsman Glasses, no more getting flashed as Corpsman!

Unkn0wnGh0st333 committed:
Corpsman Glasses

Added Corpsman Glasses to the game, added it to Corpsmans lockers, and added a crafting recipe for them.

Unkn0wnGh0st333 committed:
Corpsman Glasses

Added Corpsman Glasses to the game, added it to Corpsmans lockers, and added a crafting recipe for them.

deltanedas commented:
changelog is yapping make it 1 entry

deltanedas commented on Resources/Prototypes/DeltaV/Entities/Clothing/Eyes/glasses.yml:
```diff
@@ -12,3 +12,33 @@
+- type: entity
+  parent: [ClothingEyesBase, ShowSecurityIcons, ShowMedicalIcons, BaseRestrictedContraband]
+  id: ClothingEyesGlassesCorpsman
+  name: Corpsman Glasses
```
corpsman glasses, lowercase

Unkn0wnGh0st333 commented:
> changelog is yapping make it 1 entry

my baaaad

deltanedas commented on Resources/Prototypes/DeltaV/Entities/Clothing/Eyes/glasses.yml:
```diff
@@ -12,3 +12,33 @@
         Caustic: 0.85
+
+- type: entity
+  parent: [ClothingEyesBase, ShowSecurityIcons, ShowMedicalIcons, BaseRestrictedContraband]
```
if you are going to use contraband use base security contraband

Unkn0wnGh0st333 replied:
![image](https://github.com/user-attachments/assets/475a3c7c-0e1b-45f8-82c1-6d4fca3d7fb7)
The normal security glasses also use the "Restricted Contraband"

Unkn0wnGh0st333 replied:
there is also no "BaseSecurityContraband"

deltanedas commented on Resources/Prototypes/DeltaV/Entities/Clothing/Eyes/glasses.yml:
```diff
@@ -12,3 +12,33 @@
+  parent: [ClothingEyesBase, ShowSecurityIcons, ShowMedicalIcons, BaseRestrictedContraband]
+  id: ClothingEyesGlassesCorpsman
+  name: Corpsman Glasses
+  description: Security glasses designed for the Corpsman for medical and security needs. Now with a cool blue hue*
```
`*`

deltanedas commented on Resources/Prototypes/DeltaV/Entities/Clothing/Eyes/glasses.yml:
```diff
@@ -12,3 +12,33 @@
+  description: Security glasses designed for the Corpsman for medical and security needs. Now with a cool blue hue*
+  components:
+  - type: Sprite
+    sprite: Clothing/Eyes/Glasses/corpsglasses.rsi
```
move the sprite to DeltaV folder and update everything

deltanedas commented on Resources/Prototypes/Recipes/Construction/clothing.yml:
```diff
@@ -108,6 +108,17 @@
   icon: { sprite: Clothing/Eyes/Glasses/secglasses.rsi, state: icon }
   objectType: Item
 
+- type: construction
```
deltav folder

deltanedas commented on Resources/Prototypes/Recipes/Construction/Graphs/clothing/glasses_corpshud.yml:
```diff
@@ -0,0 +1,25 @@
+- type: constructionGraph
```
deltav folder

deltanedas commented on Resources/Prototypes/tags.yml:
```diff
@@ -674,6 +674,9 @@
 - type: Tag
   id: HudSecurity
 
+- type: Tag
```
deltav folder

Unkn0wnGh0st333 committed:
Corpsman Glasses

Added Corpsman Glasses to the game, added it to Corpsmans lockers, and added a crafting recipe for them.

Unkn0wnGh0st333 commented:
woops forgot one thing

Unkn0wnGh0st333 committed:
Corpsman Glasses

Added Corpsman Glasses to the game, added it to Corpsmans lockers, and added a crafting recipe for them.
